data/json/items/melee/bludgeons.json

```diff
@@ -361,6 +380,18 @@
     "cutting": 6,
     "flags": [ "SHEATH_KNIFE", "CONDUCTIVE", "FRAGILE_MELEE" ],
     "qualities": [ [ "BUTCHER", -18 ] ]
+  },
+    {
+    "type": "recipe",
```

Comment:
I think this should be an actual item entry, not a recipe entry and moved to `swords_and_blades.json`

Reply:
that was a misclick on my part.